src/EnergyPlus/BaseboardElectric.hh

```diff
@@ -125,7 +127,7 @@ namespace BaseboardElectric {
 
     void clear_state();
 
-    void SimElectricBaseboard(EnergyPlusData &state, std::string const &EquipName, int const ActualZoneNum, int const ControlledZoneNum, Real64 &PowerMet, int &CompIndex);
+    void SimElectricBaseboard(EnergyPlus::EnergyPlusData &state, std::string const &EquipName, int const ActualZoneNum, int const ControlledZoneNum, Real64 &PowerMet, int &CompIndex);
```

Comment:
Accompanying that change, whenever we declare it as an argument type, it needs to be declared properly with `EnergyPlus::EnergyPlusData`.  This goes with the other lower-level types you may use: `EnergyPlus::ExteriorEnergyUseData`.